e2e_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"github.com/konveyor/windup-shim/pkg/conversion"
+	"github.com/konveyor/windup-shim/pkg/windup"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func Test_convertRule(t *testing.T) {
+
+	rulesets := []windup.Ruleset{}
+	ruleTests := []windup.Ruletest{}
+
+	err := filepath.WalkDir("./data", WalkXML("./data/test-ruleset.xml", &rulesets, &ruleTests, false))
+	if err != nil {
+		fmt.Println(err)
+	}
+	_, err = conversion.ConvertWindupRulesetsToAnalyzer(rulesets, "./data/test-ruleset.xml", "./test-rule-output", true, false)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer func() {
+		err := os.RemoveAll("./test-rule-output")
```

Comment:
Let's add this to .gitignore just in case

Reply:
Solved in 37d17f5631418c7547e5becc7b0f32e4aa7e6098